tests/test_learning_k_means.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit tests for landmarks space.
+"""
+
+import geomstats
+import geomstats.backend as gs
+import geomstats.tests
+
+from geomstats.geometry import hyperbolic_space
+from geomstats.geometry import hypersphere
+from geomstats.learning.k_means import KMeans
+
+
+class TestLearningKMeans(geomstats.tests.TestCase):
```

Comment:
Should be:
```
class TestRiemannianKMeansMethods
```
(once you have renamed KMeans to RiemannianKMeans)
to be consistent with the naming in the tests

https://github.com/geomstats/geomstats/blob/master/tests/test_euclidean_space.py#L12